### ISSUE TITLE
Spycam Rules of use.

### ISSUE THREAD

**Issue by Kevbroseph** [echoroleplay/feedback#356]
**Suggestion Description**
Add rules and IC laws for the use of the Spycam.

**Reason**
I know we all are enjoying the spycams. I know I am. But I can foresee them being an issue for power gaming.
I feel that people will start using them to spy on crime and tax locations and a number of other things that could pose an issue.

**Additional context**
I think IC law can be handled with RP such as concern for privacy, warrants to place them, illegal to mount them outside your property, etc.
Which I personally Plat to attempt.

**Comment by Dancien** [comment]
I agree. Also some other suggestions. Make them more expensive than $100 and allow other people to destroy them if they see them.

**Comment by Kevbroseph** [comment]
Well, if the intention was to allow them to survive indefinitely, then for sure make them like 1k.
If only through one storm unless you retrieve it, thn I'd say keep the current price.

**Comment by Kevbroseph** [comment]
Literally just watched a stream of PD talking about how they put one down at a chop spot.
That's ridiculous.

**Comment by MamaPrince** [comment]
PD, IC is taking appropriate steps to address concerns for using in crime areas. But My opinion, they need to be nerfed is some way for the knowledge I have so far. 

1. They never expire, so what's stopping someone from placing a ton around properties or other areas to watch everywhere. Would make raids and PD movement harder as well as other civs for doing crime. 
2. Reports of people saying the are still accessible after storm and being invisible
3. Expectation of privacy. There does need to be some sort of stipulations on these being placed (that's coming from a cop/detective)
4. They are way to cheap. You could buy these in bulk and place them all over
5. There's no way to track who the cameras belong to. They should have some sort of serial number that we as cops/doj can get the information through a subpoena

**Comment by SuspiciouslyStickySock** [comment]
Add the watching of the camera into an app located within Laptop/Phone, i feel like it is a weird to just use /cameras.

**Comment by Dancien** [comment]
So technically electronic surveillance is classified as a search under the 4th Amendment and is therefore required to have a warrant much like a search warrant for a warehouse etc.

**Comment by MamaPrince** [comment]
> 

Unless we are a one party consent state (don’t need other parties permission to record video/audio). In public there would be no expectation of privacy especially if the camera is placed on public property
